### ISSUE TITLE
tldr miss TE insertion

### ISSUE THREAD

**Issue by airichli** [adamewing/tldr#41]
Hi,

Thanks for developing tldr software, which is convenient and reliable. However, I found in few cases, tldr seems to miss TE insertion. For example, I have 8 samples, tldr can identify the same TE insertion in all 8 samples, but when I merged all 8 samples into one, tldr could not output this TE insertion information. I also extract the reads around this insertion sites (small data size) to perform tldr and I got the same insertion results with individual sample (all have the same TE insertion) and merged sample (missing TE insertion).

My command line is:
tldr -b $bamfile -e $TE_ref_fa  -r $genome_fa.$chrom.fa  --color_consensus -p 2 --detail_output --extend_consensus 1000

Thank you for your time and consideration. Looking forward to hearing from you.

Best,
Aifu
